src/VersionTool/Package.cs

```diff
@@ -1,9 +1,16 @@
-﻿namespace VersionTool
+﻿using System;
+
+namespace VersionTool
 {
-    public class Package
+    public class Package : IComparable<Package>
```

Comment:
Doesn't really make sense to stick `IComparable` here since it's not an intrinsic behavior of `Package`. Just do the comparison at the place you sort.